integrations/nlu-snips/src/SnipsNlu.ts

```diff
@@ -68,7 +68,12 @@ export class SnipsNlu extends NluPlugin<SnipsNluConfig> {
         nluData.entities = {};
       }
 
-      nluData.entities[slot.slotName] = { key: slot.rawValue, value: slot.value.value };
+      nluData.entities[slot.slotName] = {
+        id: slot.rawValue,
```

Comment:
According to [this documentation](https://snips-nlu.readthedocs.io/en/latest/tutorial.html#parsing), I think it should be the following:

```typescript
id: slot.value.value,
resolved: slot.value.value,
value: slot.rawValue,
native: slot,
```

Reply:
I changed it accordingly, thank you.